srcpkgs/jaaa/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'jaaa'
+pkgname=jaaa
+version=0.9.2
+revision=1
+build_wrksrc="source"
+build_style=gnu-makefile
+hostmakedepends="pkgconf"
```

Comment:
pkg-config

Reply:
i'm sure it is pkgconf and not pkg-config

Reply:
it is pkg-config

Reply:
but when i replace pkgconf with pkg-config the build terminates with "make: pkgconf: Command not found" :(

Reply:
sed the Makefiles to make it use pkg-config instead of pkgconf, pkg-config is the standard name that is used everywhere.

Pkgconf is just the most popular provider of pkg-config, hate-able that people are ignoring compatibility specially one that has 0 cost for keeping.

Reply:
This is a serious bug upstream